packages/js/components/src/date-time-picker-control/date-time-picker-control.tsx

```diff
@@ -68,6 +69,7 @@ export const DateTimePickerControl = forwardRef(
 			help,
 			className = '',
 			onChangeDebounceWait = 500,
+			popoverProps = {},
```

Comment:
Since the Dropdown component has [a known issue](https://github.com/woocommerce/woocommerce/issues/41297) for date pickers, I added this prop to increase flexibility. In this specific case, we want the component rendered not inline.